src/aws_signature.app.src

```diff
@@ -1,14 +1,13 @@
 {application, aws_signature,
  [{description, "Request signature implementation for authorizing AWS API calls"},
-  {vsn, "0.3.1"},
+  {vsn, "git"},
```

Comment:
I think this needs to `git` atom and we need to configure the rebar plugin, no? Though I would keep it as is, it's always a good idea to grep the current version to make sure all places are updated, and we don't release often either :)

Reply:
> I think this needs to `git` atom and we need to configure the rebar plugin, no? Though I would keep it as is, it's always a good idea to grep the current version to make sure all places are updated, and we don't release often either :)

🤔 not according to: [aws-erlang - aws.app.src#3](https://github.com/aws-beam/aws-erlang/blob/master/src/aws.app.src#L3) which I did recently 🤔 I'll double check the rebar plugin but I don't remember making any changes on that front.

Reply:
Nevermind then, I just was curious where is it handled and it lead to the plugin, but if it's built-in then all good :D

Reply:
`git` instructs rebar3 to attempt to parse a tag and use that as the version string of a project. Sometimes it is even smart/correct about it!